src/pybind/mgr/tox.ini

```diff
@@ -161,6 +162,7 @@ modules =
 commands =
     flake8 --config=tox.ini {posargs} \
       {posargs:{[testenv:flake8]modules}}
+    bash -c 'test $(git ls-files cephadm | grep ".py$" | grep -v tests | xargs grep "docker.io" | wc -l) == 12'
```

Comment:
what's the reason behind testing how many times docker.io appears in the codebase (and why are we assuming it should stay at this particular number, https://github.com/ceph/ceph/pull/43901 for example would change it)?

Reply:
I'd like to allow folks to change this number in their downstreams in order to make sure no new references to docker.io are creeping in unnoticed.

Reply:
makes sense